Libraries/LibWeb/DOM/Slottable.cpp

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2023, Tim Flynn <trflynn89@serenityos.org>
+ * Copyright (c) 2025, Shannon Booth <shannon@serenityos.org>
```

Comment:
Nit: You use different emails on the two files you modified, that seems unintentional.

Reply:
whoops, I certainly don't have a ladybird email, that was a copy paste mistake! thanks